### PR TITLE
rename existing mpi to horovod

### PR DESCRIPTION
**Pull request by cheyang**
In order to support MPI operator, we have to rename the current MPI implementation to Horovod.

### CODE REVIEW

File: cmd/arena/commands/submit.go [hunk 1]
```diff
@@ -133,9 +133,9 @@ var (
 	submitLong = `Submit a training job.
 
 Available Commands:
-  tfjob,tf          Submit a TFJob.
-  mpijob,mpi        Submit a MPIJob.
-  standalonejob,sj  Submit a standalone Job.
+  tfjob,tf             Submit a TFJob.
+  horovod,hj        Submit a MPIJob.
```
Comment (wsxiaozhang) [inline cmd/arena/commands/submit.go]: `horovod,horovod     Submit a HorovodJob.`

Comment (wsxiaozhang) [inline cmd/arena/commands/submit.go]: `horovodjob,hj    Submit a HorovodJob.`

File: cmd/arena/commands/submit_horovod.go [hunk 2]
```diff
@@ -14,13 +14,11 @@ func NewSubmitHorovodJobCommand() *cobra.Command {
...
 		submitArgs submitHorovodJobArgs
 	)
-	// TODO(cheyang): set this mode as mpijob for short time
-	submitArgs.Mode = "mpijob"
 
 	var command = &cobra.Command{
-		Use:     "mpijob",
-		Short:   "Submit MPIJob as training job.",
-		Aliases: []string{"mpi"},
+		Use:     "horovod",
```
Comment (wsxiaozhang) [inline cmd/arena/commands/submit_horovod.go]: `Use:    "horovodjob",`

**Comment by cheyang** [comment]
@wsxiaozhang PTAL.

**Pull request merged**